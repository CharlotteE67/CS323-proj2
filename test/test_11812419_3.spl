int get_int(){
    return 0;
}

float get_float(){
    return -1.0;
}

int main(){
    int x,y;
    float a,b;

    x = get_int();
    get_float() = y;
    b = get_int();

    x = a = y;

}