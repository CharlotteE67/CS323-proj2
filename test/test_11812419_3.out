Error type 6 at Line 14: value can not be assigned.
Error type 5 at Line 15: unmatching types appear at both sides of the assigment operator.
Error type 5 at Line 17: unmatching types appear at both sides of the assigment operator.
Error type 5 at Line 17: unmatching types appear at both sides of the assigment operator.